github/enterprise_custom_properties.go

```diff
@@ -0,0 +1,189 @@
+// Copyright 2025 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"context"
+	"fmt"
+)
+
+// EnterpriseCustomPropertySchema represents the schema response for GetEnterpriseCustomPropertiesSchema.
+type EnterpriseCustomPropertySchema struct {
+	// An ordered list of the custom property defined in the enterprise.
+	Properties []*CustomProperty `json:"properties,omitempty"`
+}
+
+// EnterpriseCustomPropertiesValues represents the custom properties values for an organization within an enterprise.
+type EnterpriseCustomPropertiesValues struct {
+	// The Organization ID that the custom property values will be applied to.
+	OrganizationID *int64 `json:"organization_id,omitempty"`
+	// The names of organizations that the custom property values will be applied to.
+	OrganizationLogin *string `json:"organization_login,omitempty"`
+	// List of custom property names and associated values to apply to the organizations.
+	Properties []*CustomPropertyValue `json:"properties,omitempty"`
+}
+
+// EnterpriseCustomPropertyValuesRequest represents the request to update custom property values for organizations within an enterprise.
+type EnterpriseCustomPropertyValuesRequest struct {
+	// The names of organizations that the custom property values will be applied to.
+	// OrganizationLogin specifies the organization name when updating multiple organizations.
+	OrganizationLogin []string `json:"organization_login"`
+	// List of custom property names and associated values to apply to the organizations.
+	Properties []*CustomPropertyValue `json:"properties"`
+}
+
+// GetEnterpriseCustomPropertySchema gives all organization custom property definitions that are defined on an enterprise.
+//
+// GitHub API docs: https://docs.github.com/enterprise-cloud@latest/rest/enterprise-admin/custom-properties-for-orgs#get-organization-custom-properties-schema-for-an-enterprise
+//
+//meta:operation GET /enterprises/{enterprise}/org-properties/schema
+func (s *EnterpriseService) GetEnterpriseCustomPropertySchema(ctx context.Context, enterprise string) (*EnterpriseCustomPropertySchema, *Response, error) {
```

Comment:
```suggestion
func (s *EnterpriseService) GetOrganizationCustomPropertySchema(ctx context.Context, enterprise string) (*EnterpriseCustomPropertySchema, *Response, error) {
```
As we're in the enterprise service we don't need `Enterprise` in the name, but these are organization custom properties according to the docs. The same applies to the other methods.